src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php

```diff
@@ -68,6 +68,11 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
                     $attributeMetadata->setSerializedName((string) $attribute['serialized-name']);
                 }
 
+                if (isset($attribute->{'serialized-path'})) {
```

Comment:
Seems like nested attributes are written using snake case instead (see `denormalization_context` for example)